canonical/jp.ubuntu.com
# Refresh landing page

## Done

* Apply webpage refresh based on design and copy docs
* Redesign navigation, footer and takeovers
* Design: https://www.figma.com/design/tI8YK4wQnKiWLAxdX1fWnB/24.10-jp.ubuntu.com?node-id=53-1436&t=VwJrN9XBr3URiSLE-0
* Copy docs: https://docs.google.com/document/d/1J4zHHJ19ueONPtgj9oHTWGvWRX5JTmHy8VuGy4mygp8/edit?tab=t.0#heading=h.ruj033ajm4pj

## QA

- Go to [demos landing page](https://jp-ubuntu-com-478.demos.haus/)
- Check that design and info matches up with Figma and copy docs

## Issue / Card

Fixes [WD-11638](https://warthogs.atlassian.net/browse/WD-11638)

## Screenshots

[if relevant, include a screenshot]


[WD-11638]: https://warthogs.atlassian.net/browse/WD-11638?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
